test/browser/index.js

```diff
@@ -28,6 +28,9 @@ require('./components');
 // test layers
 require('./layer-tests');
 
+// test formatters
+require('./formatters');
```

Comment:
if you only have 1 formatter test, just directly import here

`require('./formatters-tests')`

Reply:
But what if we would have additional formatters in the future?